app/precompiles.go

```diff
@@ -3,27 +3,65 @@ package app
 import (
 	"fmt"
 
+	"cosmossdk.io/core/address"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
 
-	authzkeeper "github.com/cosmos/cosmos-sdk/x/authz/keeper"
+	addresscodec "github.com/cosmos/cosmos-sdk/codec/address"
 	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
 	distributionkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
-	channelkeeper "github.com/cosmos/ibc-go/v8/modules/core/04-channel/keeper"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	bankprecompile "github.com/cosmos/evm/precompiles/bank"
+	"github.com/cosmos/evm/precompiles/bech32"
+	distprecompile "github.com/cosmos/evm/precompiles/distribution"
+	govprecompile "github.com/cosmos/evm/precompiles/gov"
+	ics20precompile "github.com/cosmos/evm/precompiles/ics20"
+	"github.com/cosmos/evm/precompiles/p256"
+	stakingprecompile "github.com/cosmos/evm/precompiles/staking"
+	erc20Keeper "github.com/cosmos/evm/x/erc20/keeper"
+	transferkeeper "github.com/cosmos/evm/x/ibc/transfer/keeper"
+	evmkeeper "github.com/cosmos/evm/x/vm/keeper"
+	channelkeeper "github.com/cosmos/ibc-go/v10/modules/core/04-channel/keeper"
 	"github.com/ethereum/go-ethereum/common"
-	bankprecompile "github.com/evmos/evmos/v20/precompiles/bank"
-	"github.com/evmos/evmos/v20/precompiles/bech32"
-	distprecompile "github.com/evmos/evmos/v20/precompiles/distribution"
-	govprecompile "github.com/evmos/evmos/v20/precompiles/gov"
-	ics20precompile "github.com/evmos/evmos/v20/precompiles/ics20"
-	"github.com/evmos/evmos/v20/precompiles/p256"
-	stakingprecompile "github.com/evmos/evmos/v20/precompiles/staking"
-	erc20Keeper "github.com/evmos/evmos/v20/x/erc20/keeper"
-	"github.com/evmos/evmos/v20/x/evm/core/vm"
-	transferkeeper "github.com/evmos/evmos/v20/x/ibc/transfer/keeper"
-	stakingkeeper "github.com/evmos/evmos/v20/x/staking/keeper"
+	"github.com/ethereum/go-ethereum/core/vm"
 	"golang.org/x/exp/maps"
 )
 
+type Optionals struct {
```

Comment:
What are this optionals? I don't see them in the guide https://evm.cosmos.network/docs/v0.4.x/documentation/integration/evm-module-integration#step-6%3A-create-precompiles-configuration